internal/environment/environment.go

```diff
@@ -59,6 +59,9 @@ func (p Paths) BasePath() string { return p.base }
 // e.g. {BasePath}/index/
 func (p Paths) IndexPath() string { return filepath.Join(p.base, "index") }
 
+// DefaultIndexPath returns the base directory where the default plugin index repository is cloned.
+func (p Paths) DefaultIndexPath() string { return filepath.Join(p.base, "index", "default") }
```

Comment:
I feel like we don't need a method like this. Something like we eventually need to refactor this to `IndexPath(name string)`.

The method above (`IndexPath()`) ideally should not be necessary (except for some tests, e.g. below).

Reply:
Ya that makes sense, I'll remove it since this will check for the `.git` directory.

Reply:
SGTM.